_docs/templates/email-template.md

```diff
@@ -66,56 +95,86 @@ followed by `.rackspace.com`.
 number**, no other numerical information should be included in email subject
 lines.
 
+{% endcolumn %}
+
+</div>
+
 ## Email content
 
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
 ### Alert status
 
 Alert states are used to gain the users attention when there is action required
 on the customers part or when sending important impacting notifications. If
 there is no alert status and the severity of the message is normal, an alert
 banner should **not** be included.
 
-<div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 - Ticket Status:
+
     - Emergency – Address in Subject line and with Helix Alert bar
     - Urgent – Address in the Subject line and with simple alert bar
     - Normal – No specification needed in email
+
 - Notification Status:
+
     - Urgent – Address in Subject line and simple alert bar
     - Normal – No specification needed
+
 - Server Status: (No Banner Needed)
+
     - Status Code – [View specifications](https://rackerlogin.fedsso.rackspace.com/nidp/app?target=https%3A%2F%2Frackerlogin.fedsso.rackspace.com%2Fnidp%2Fsaml2%2Fidpsend%3Fid%3Donewiki%26%26SAMLRequest%3DlVJdS%252BtAFPwrYR98a9JEQVmblFIRBAWpHw%252B%252BbXZPm73JfrhnU%252F35nqYGgngvvbCQkD0zc2Ymi%252BWn%250A6ZI9BNTOlixP5ywBK53Sdleyl%252Bfb2RVbVgsUpvN81cfGbuC9B4wJ4Szy4aJkfbDcCdTIrTCAPEr%252B%250AtHq450U65z646KTrWLJChBBJaO0s9gbCE4S9lvCyuS9ZE6NHnmXOQhqEbNELCal0JvNdv9MWMwLv%250AO4jZQRPRseSG9tBWxGH1EX%252FAQugcQdItKBr8QWe18gNHkdEbglVLrUqS%252FdCtPhPGX7Pk7oYITV3%252F%250AUaquGzqgW18bLXZOelWDadvG7Kza1rqhacQe7ixGYWPJinl%252BOZvns%252BLiucj5%252BRWd9Dy%252FfGPJ6xgz%250AxcLGUAdwOD1OMYbIqv%252BObJFNNY8bFJ5%252FVwpqKJjKifA5Lbg4ueG1M14EjQeTRlttevNtlE%252B51x25%250A2MB2onGy6X%252BOSS4P1PT5kR4fLqij5V%252FVq%252BPd3wKoxrSmv331BQ%253D%253D%26RelayState%3D%252Fpages%252Fviewpage.action%253Ftitle%253DCloud%252BServers%252BStatus%252BSpecification%2526spaceKey%253Dreach%26TARGET%3D%252Fpages%252Fviewpage.action%253Ftitle%253DCloud%252BServers%252BStatus%252BSpecification%2526spaceKey%253Dreach)
     - Remote Observations
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:""] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Urgent%20Alert.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Urgent%20Alert.svg)
 {% endfigure %}
 {% endcolumn %}
+
 </div>
 
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
 {% figure [caption:"Alert banner typography, color, and spacing"] [class:"image bg-light border"] %}
   ![]({{site.cdn_url}}/img/layout/email-templates/Alert%20Specs.svg){:width=60%}
```

Comment:
More spacing oddities (`Robo to`, `#   FAFAFA`) and spelling fix needed:
<img width="273" alt="screen shot 2017-11-20 at 11 04 52 am" src="https://user-images.githubusercontent.com/1538692/33031052-b67495e6-cde2-11e7-8cc5-1d146e662cef.png">

Also, can we drop the width modifier, or is it needed here?